src/container/GeoWebSystem/GWS.jsx

```diff
@@ -132,16 +132,19 @@ const GWS = () => {
             );
         }
     }
-
-
-    return(
-        <div>
-            <TitleBar accessGps={accessGps} />
-
-            { loading ? <GWLoader/> : <GeoWeb /> }
-
-            {/*Display Mock Data*/}
-            {/* <div style={{position: "absolute", top: '20%', color: 'white', width:'50%'}}>
+  };
```

Comment:
This line might be a typo? Without it should pass the build job.

Reply:
Yes thanks for catching that, just fixed